int16 id
nav_msgs/Path path